### DISCUSSION TITLE
关于主页关键词屏蔽或过滤

### DISCUSSION THREAD

**Discussion by 0x00Finch** [the1812/Bilibili-Evolved#2087] [功能建议 / Ideas]
## 希望增加首页关键词过滤或屏蔽功能（最好包括正则）

**Comment by the1812** [comment]
简化首页下好像是支持的, 根据动态过滤器里的关键词

**Reply by 0x00Finch** [reply]
那个只是动态里过滤，我说的是首页

**Reply by the1812** [reply]
我就是说首页啊, 开了简化首页时就能过滤, 只是设置还是在动态那边设置

**Comment by 0x00Finch** [comment]
![132123](https://user-images.githubusercontent.com/26269474/126923429-24c57a9e-aa52-45a3-9fff-6b20b04c30be.jpg)
![12312312313123](https://user-images.githubusercontent.com/26269474/126923432-23ad20f3-3bf2-4c1a-b248-2de2ad39afe6.jpg)

> 自己看

**Reply by 0x00Finch** [reply]
难道是我过滤的方式不对？

**Reply by the1812** [reply]
你这是原版首页, 我说了是简化首页

**Reply by 0x00Finch** [reply]
我试试看，应该是没开

**Reply by 0x00Finch** [reply]
> 你这是原版首页, 我说了是简化首页

>> 感谢

**Comment by 0x00Finch** [comment]
@the1812 
![1894841894189](https://user-images.githubusercontent.com/26269474/126931284-2ecd6e57-4194-4514-8560-4a8bfe0bd944.jpg)
这是否是简化首页的bug呢？

**Reply by 0x00Finch** [reply]
现在是简化首页了，屏蔽功能还是不好使

**Reply by the1812** [reply]
是不是热门视频那里的, 我现在只做了下面那行动态的过滤, 需要的话我可以把热门视频也过滤了

**Reply by 0x00Finch** [reply]
> 是不是热门视频那里的, 我现在只做了下面那行动态的过滤, 需要的话我可以把热门视频也过滤了

如果能做的话，那太感谢了！

**Comment by 0x00Finch** [comment]
@the1812 所以大佬，什么时候能加上热门视频的关键词过滤？

**Reply by RingingVolcano** [reply]
别 着 急
最近主要在做从V1到V2的迁移 新功能优先级不太高

**Reply by 0x00Finch** [reply]
好的

**Reply by the1812** [reply]
下个预览版
